VideoApi/VideoApi.UnitTests/Controllers/Conference/GetConferencesForAdminByHearingRefIdTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using VideoApi.Contract.Responses;
+using VideoApi.DAL.Queries;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.UnitTests.Controllers.Conference
```

Comment:
I would encourage API Integration tests first and then save unit tests for the scenarios that are difficult to test without a stub.

Reply:
done